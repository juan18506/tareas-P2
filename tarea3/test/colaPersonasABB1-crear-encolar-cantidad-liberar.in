crearCola

# imprimo la cantidad de elementos de la cola, que debe ser 0
imprimirCantidadCola

# Creo arbol con Persona 4 y 5
crearArbolPersonas

crearPersona 5 11 Maria
insertarPersonaArbolPersonas

crearPersona 4 55 Raul
insertarPersonaArbolPersonas

# Encolo el arbol con Persona 4 y 5
encolar

# imprimo la cantidad de elementos de la cola, que debe ser 1
imprimirCantidadCola

# obtengo el frente de la cola, que es el árbol con las Persona 4 y 5
obtenerFrenteCola

# libero la memoria asociada a la cola
liberarCola

# imrpimo el arbol con las Persona 4 y 5
imprimirArbolPersonas

# libero el árbol
liberarArbolPersonas

Fin