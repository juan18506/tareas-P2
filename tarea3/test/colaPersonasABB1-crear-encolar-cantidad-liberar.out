1>2>#  imprimo la cantidad de elementos de la cola, que debe ser 0.
3>La cantidad de arboles de personas en la cola es: 0.
4>#  Creo arbol con Persona 4 y 5.
5>6>7>8>9>10>#  Encolo el arbol con Persona 4 y 5.
11>12>#  imprimo la cantidad de elementos de la cola, que debe ser 1.
13>La cantidad de arboles de personas en la cola es: 1.
14>#  obtengo el frente de la cola, que es el árbol con las Persona 4 y 5.
15>Persona 4:  Raul, 55 años
Persona 5:  Maria, 11 años
16>#  libero la memoria asociada a la cola.
17>18>#  imrpimo el arbol con las Persona 4 y 5.
19>Persona 4:  Raul, 55 años
Persona 5:  Maria, 11 años
20>#  libero el árbol.
21>22>Fin.
